android/java/org/chromium/chrome/browser/BraveRewardsHelper.java

```diff
@@ -118,11 +117,12 @@ public static boolean hasRewardsOnboardingModalShown() {
     }
 
     public static boolean shouldShowRewardsOnboardingModalOnDay4() {
```

Comment:
where do we use that function? I cannot find any calls to it.

Reply:
It looks like the call to it was removed here https://github.com/brave/brave-core/pull/11768/files#diff-448df90d6aa5a20513ca5dba42a4e428dfd22f05577b904f8fe7986c40ab90faL560. I can remove the function then.

Reply:
cc: @deeppandya should I remove or is it meant to be called?

Reply:
@SergeyZhukovsky we need to remove left over code in a separate issue. i will handle it in https://github.com/brave/brave-browser/issues/26171